src/main/java/com/semmle/jira/addon/workflow/LgtmTransitionNotificationFunction.java

```diff
@@ -45,14 +48,20 @@
   @SuppressWarnings("rawtypes")
   public void execute(Map transientVars, Map args, PropertySet ps) throws WorkflowException {
 
-    String configKey = "webhook"; // TODO get config key from custom field
+    Transition transition = getTransitionParam(args);
+    MutableIssue issue = getIssue(transientVars);
+
+    CustomField customField =
```

Comment:
Probably best to retrieve the field's ID from the configuration instead of dynamically resolving it by name.